api/app/Rules/SkillNotUsedByActivePool.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Rules;
+
+use App\Models\Pool;
+use Carbon\Carbon;
+use Closure;
+use Database\Helpers\ApiErrorEnums;
+use Illuminate\Contracts\Validation\ValidationRule;
+
+class SkillNotUsedByActivePool implements ValidationRule
+{
+    /**
+     * Run the validation rule.
+     */
+    public function validate(string $attribute, mixed $value, Closure $fail): void
+    {
+        // validation fails if skill is in use by an active pool
+        $skillId = $value;
+        $activePools = Pool::where('published_at', '<=', Carbon::now()->toDateTimeString())
+            ->where('closing_date', '>', Carbon::now()->toDateTimeString())
```

Comment:
Feels like `currentlyActive` or `currentlyOpen` should be a scope

Reply:
Yeah, definitely
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7762/commits/39e1b082ceaea1013f475b787df0101484128abf and tested with https://github.com/GCTC-NTGC/gc-digital-talent/pull/7762/commits/b46cb0fcf3ce168fcc108ea996892665f6d540ce